walkthroughs/week-4-libraries/maps/examples/marker-storage/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -35,7 +35,12 @@
 @WebServlet("/markers")
 public class MarkerServlet extends HttpServlet {
 
-  /** Responds with a JSON array containing marker data. */
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
More uniqueness needed?

Some reviewers will pedantically identify every place a comment applies, that's not my way. I think it's generally considered good practice for coders to consider whether there are other places in which they can apply the point the reviewer is making.

Reply:
That sounds like a good approach to me :)

Reply:
All of them should be unique now